ql/src/java/org/apache/hadoop/hive/ql/exec/FunctionRegistry.java

```diff
@@ -32,6 +32,7 @@
 import java.util.regex.Pattern;
 import java.util.regex.PatternSyntaxException;
 
+import org.apache.hadoop.hive.ql.udf.*;
```

Comment:
please avoid wildcard imports

Reply:
Removed wildcard import and imported only necessary newly added UDF classes